docs/src/pages/docs/reference/rag/rerank.mdx

```diff
@@ -57,8 +57,8 @@ const rerankedResults = await rerank(
     },
     {
       name: "model",
-      type: "LanguageModelV1",
-      description: "The language model to use for reranking",
+      type: "MastraLanguageModel",
+      description: "The Mastra Language Model to use for reranking",
```

Comment:
Docs might not need to be changed, or I guess we need to add somewhere what MastraLanguageModel is